macros/REST_Track_PlotAlphaAna.C

```diff
@@ -0,0 +1,457 @@
+//
+//
+//
+#include <TRestRun.h>
+#include <TRestTrackAlphaAnalysisProcess.h>
+#include <TRestDetectorReadout.h>
+#include <TRestTrackEvent.h>
+#include <TRestAnalysisTree.h>
+#include <TRestDetectorHitsEvent.h>
+#include <TRestRawSignalEvent.h>
+
+TH2F *trackX;
+TH2F *trackY;
+
+TH2F *hitsX;
+TH2F *hitsY;
+
+std::vector<TGraph *> pulsesX;
+std::vector<TGraph *> pulsesY;
+
+const int offset=10;//bins
+
+
+TPad *DrawTrack ( TRestTrackEvent *trackEvent, const TVector3 &origin, const TVector3 &end, const TVector3 &max, const TVector3 &min, const TVector3& nBins) {
```

Comment:
Perhaps better to implement at `TRestTrackEvent`

Reply:
I understand, but anyway it could be implemented at TRestTrackEvent and the macro would just call to this method?

See for example `SetBoundaries` method at `TRestDetectorHitsEvent`.

Of course, those methods could be even further promoted to a higher class, it would be good to have dedicated drawing classes for hits and for time signals that centralize that kind of staff.